src/Mobs/PassiveMonster.cpp

```diff
@@ -133,7 +138,7 @@ void cPassiveMonster::Tick(std::chrono::milliseconds a_Dt, cChunk & a_Chunk)
 
 				virtual bool Item(cEntity * a_Entity) override
 				{
-					// if we're the same species as someone around and they dont have a partner, swipe right
+					// if we're the same species as someone around and they don't have a partner, swipe right.
```

Comment:
I, ah, have something in mind, but maybe not.

Instead: what sort of tinder?
![Tinder](http://orig02.deviantart.net/3aba/f/2011/324/c/e/minecraft_tcg_090__flint_and_tinder_by_acaroa-d4gtqph.jpg)